include/gnc_constants.hpp

```diff
@@ -0,0 +1,52 @@
+//
+// include/gnc_constants.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+/* - NOTICE - Values in this file should be seldomly edited and have hidden
+ *            interdependancies. */
+
+#ifndef PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+#define PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+
+#include "lin.hpp"
+
+namespace gnc {
+namespace constant {
+
+/** Initial GPS week number that serves as the 'PAN epoch'. Some of the
+ *  constants below depend on this number. */
+constexpr static unsigned int init_gps_week_number = 2045;
```

Comment:
Can you add GPS time of week and nanoseconds here as well?

Reply:
Yep for sure we will be adding more constants. How we handle time specifically is something I think I'm lagging on. It's on my list of potential topics for Thursday.

Reply:
Since GPS epoch is a specific constant that FSW requires I think it'd be good if we can get it in on this PR